protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -385,14 +385,18 @@ service TradingDataService {
 
   // List reward summaries
   //
-  // Get a list of reward summaries that match the provided criteria. If no filter is provided, all reward summaries will be returned.
+  // ListRewardSummaries returns a list of rewards for a given single party ID and asset ID.
```

Comment:
We need to keep this line starting with "Get a list of reward summaries..." because Tan just went around and standardised them all to start like this!

I think whats missing is how it differs from `ListEpochRewardSummaries`. I'm guessing `ListRewardSummaries` is the total rewards over all epochs? If thats the case then we should say that here.

Reply:
Thanks, corrected!